src/json-rpc/send-request.ts

```diff
@@ -25,7 +26,7 @@ export async function sendRequest<TParams, TResult>(
 ): Promise<TResult> {
   // Generate a unique request ID using the timestamp and a random number
   const randomNumber = Math.floor(Math.random() * 1000);
-  const requestID = `req_${Date.now()}${randomNumber}`;
+  const requestID = `req_` + bs58.encode(new Buffer(`${Date.now()}${randomNumber}`));
```

Comment:
LGTM, though I would probably do:
```
const seed = `${Date.now()}${randomNumber}`;
const requestID = `req_${bs58.encode(new Buffer(seed))}`;
```

Reply:
I echo Seth's comments about using a single type of string interpolation.  But other than that, LGTM.

Reply:
Good feedback. Updated in https://github.com/ShipEngine/shipengine-js/pull/116/commits/998b2bfe4bb3804fd9619247b35cd143650705e3